config.yaml

```diff
@@ -72,11 +72,16 @@ params:
         company: 2i2c
         details: |1-
           - **Designed, deployed, and maintained resilient cloud infrastructure** across AWS, Azure, and GCP, including production-grade Kubernetes clusters.
+          - **Designed and deployed a robust Python application** to automate backup creation and management for Google Filestore instances storing critical customer data, ensuring reliable disaster recovery in production environments.
```

Comment:
I'd reword this to bring backup management and disaster recovery system to be the bolded / highlighted thing.